requirements.txt

```diff
@@ -1,11 +1,20 @@
-netaddr==0.7.19
-argparse==1.2.1
+asn1crypto==0.24.0
+bcrypt==3.1.6
+cffi==1.12.3
+configobj==5.0.6
+cryptography==2.6.1
+enum34==1.1.6
 funcsigs==1.0.2
-paramiko==1.17.6
+ipaddress==1.0.22
+mock==2.0.0
+netaddr==0.7.19
+nose==1.3.7
```

Comment:
I think we can get rid of `nose`...
I find it more clear to have an additional `requirements.txt` file in `test` which lists all the packages required to run the tests. Thus we keep the base `requirements.txt` clean.

Reply:
Good point, I'll remove it.

Reply:
done in 414e922, same for ``mock`` :)

Reply:
lgtm